src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/SceneExploreDiffFlameGraph.tsx

```diff
@@ -54,18 +59,42 @@ export class SceneExploreDiffFlameGraph extends SceneObjectBase<SceneExploreDiff
     profileMetricVariable.setState({ query: ProfileMetricVariable.QUERY_SERVICE_NAME_DEPENDENT });
     profileMetricVariable.update(true);
 
+    this.subscribeToEvents();
+
     return () => {
       profileMetricVariable.setState({ query: ProfileMetricVariable.QUERY_DEFAULT });
       profileMetricVariable.update(true);
     };
   }
 
+  subscribeToEvents() {
+    this._subs.add(
+      this.subscribeToEvent(EventDiffAutoSelect, () => {
+        this.autoSelectDiffRanges();
+      })
+    );
+
+    this._subs.add(
+      this.subscribeToEvent(EventDiffChoosePreset, () => {
+        this.state.presetsPicker.openSelect();
+      })
+    );
+  }
+
+  autoSelectDiffRanges() {
```

Comment:
**suggestion (non-blocking)**: Make the change a bit more explicit

At first I wasn't sure what and why was selected. Would it make sense to add some description on what "auto-select" actually means? (i.e. selecting first half of the first range and second half of the second range)

Reply:
Ok, I guess it's better to do it in `SceneComparePanel` where the logic is then